packages/timeline-state-resolver-types/src/atem.ts

```diff
@@ -81,11 +81,14 @@ export type SuperSourceBox = {
 }
 
 export interface AtemTransitionSettings {
-	// dip
 	// DVE
 	mix?: {
 		rate: number
 	}
+	dip?: {
```

Comment:
Is it possible to use mix, dip and/or wipe at the same time?

Reply:
I don't know, but I assume no since they are trying to do the same thing